cran-comments.md

```diff
@@ -21,20 +21,6 @@
 
 This is a resubmission. Thank your for the comments, below are our answers:
 
-> Please always write package names, software names and API (application
-programming interface) names in single quotes in title and description.
-e.g: --> 'Rust'
-
-> Please note that package names are case sensitive. -> in your
-description you write 'Polars' but your package name starts with a lower
-case "p" ?
-
-We have put the software and package names between single quotes. Regarding 
-"Polars", there is a distinction between "polars" (all lowercase) which is the
-name of the R package, and "Polars" (first letter uppercase) which is the name
-of the query engine that is used by "polars". This is why we use 'Polars' in the
-description.
-
```

Comment:
Why do you remove these comments? Have they seen them?

Reply:
I was wondering if reviewer did not address  unexported functions comment because there were too many comments.

Since he did not address it this time, and we likely fixed it, I removed it.

Should I add it back in?